packages/example/README.md

```diff
@@ -1,49 +1,75 @@
 # Code example
 
-This directly contains a simple, framework-independent example for Connect-Web.
-It is a web interface for [ELIZA](https://en.wikipedia.org/wiki/ELIZA), a very
-simple natural language processor built in the 1960s to represent a 
-psychotherapist. The ELIZA service is provided by us at [demo.connect.build](https://demo.connect.build).
-You can find the protocol buffer schema [on the BSR](https://buf.build/bufbuild/eliza/tree/main:buf/connect/demo/eliza/v1/eliza.proto).
+This directory contains a simple example for running a Connect-Web client with a Connect-Node server using [Express](https://expressjs.com/).
+
+The application is a web interface for a stripped-down version of [ELIZA](https://en.wikipedia.org/wiki/ELIZA), a very
+simple natural language processor built in the 1960s to represent a psychotherapist. 
 
+You can find the protocol buffer schema [on the BSR](https://buf.build/bufbuild/eliza/tree/main:buf/connect/demo/eliza/v1/eliza.proto).
 
 ## Run the example
 
-To run this example in your browser, simply run the following commands:
+You will need [Node 18](https://nodejs.org/en/download/) installed. Download the example project and install 
+its dependencies:
 
 ```shell
 curl -L https://github.com/bufbuild/connect-web/archive/refs/heads/main.zip > connect-web-main.zip
 unzip connect-web-main.zip 'connect-web-main/packages/example/*'
 
 cd connect-web-main/packages/example
 npm install
-npm run start
 ```
 
+Next, start the Connect-Node server:
+
+```shell
+npm start
+```
+
+That's it!  You should now be able to open a web browser to http://localhost:8080 and see the example running locally.
+
 ![Screenshot](README.png)
 
-## Generate code yourself
 
-If you want to use [`buf`](https://github.com/bufbuild/buf) to generate the code, 
-simply run `buf generate` in this directory. [`buf.gen.yaml`](./buf.gen.yaml) 
-contains the plugin configuration. 
+## Using Node.js as a client
+
+The file `src/client.ts` implements a CLI client that you can run in Node.js. 
 
-Of course, you can use `protoc` as well:
+```shell
+$ npm run client
+```
+
+```
+What is your name?
+> John
```

Comment:
The screenshot above uses "Donald" 😆

Reply:
Ugh.  Fixed.  Also made some fixes to the Makefile also that I noticed during build.